admin-manual/installation-setup/installation/_csv/install-landing.csv

```diff
@@ -0,0 +1,12 @@
+Intended Use,Environment,Related Documents,Notes
```

Comment:
I think that it should read instead:

    Intended use,Environment,Related documents,Notes

I don't think we have a capitalization rule but if you look around I think we're mostly using sentence case for headings and titles.

Reply:
Yay for the style guide! Updating per the cap. rule